easy-ui-react/src/Stepper/StepButton.module.scss

```diff
@@ -130,7 +129,6 @@
 
 .indicatorIncomplete {
   border: 1px solid component-token("step-button", "incomplete-border");
-  margin: calc(#{design-token("space.0.5")} - 1px) design-token("space.0");
```

Comment:
originally added this to offset the height difference from the border, but i dont think this was working as i intended